### PR TITLE
Device-management-application-review

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

**Review by sharath-jk — APPROVED** [review]
Approving changes for Edge.

File: content/device-integration/mqtt-examples-bundle/hello-mqtt-java-certificates.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ In order to follow this tutorial, check the following prerequisites:
 If you don't have a valid certificate, you can generate one for testing purposes, following the instructions below.
 
 1.  Download the scripts from the [cumulocity-examples](https://github.com/SoftwareAG/cumulocity-examples/tree/develop/mqtt-client/scripts) repository.
-2.  Create a root self-signed certificate (execute the script *00createRootSelfSignedCertificate.sh*) and upload it to your tenant. You can do it via [Device Management in the UI](/users-guide/device-management/#managing-trusted-certificates) or via [REST](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Tenant-API).
+2.  Create a root self-signed certificate (execute the script *00createRootSelfSignedCertificate.sh*) and upload it to your tenant. You can do it via [Device management in the UI](/users-guide/device-management/#managing-trusted-certificates) or via [REST](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Tenant-API).
```
Comment (BeateRixen) [inline content/device-integration/mqtt-examples-bundle/hello-mqtt-java-certificates.md:24]:
```suggestion
2.  Create a root self-signed certificate (execute the script *00createRootSelfSignedCertificate.sh*) and upload it to your tenant. You can do it via [the Device management application in the UI](/users-guide/device-management/#managing-trusted-certificates) or via [REST](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Tenant-API).
```

File: content/streaming-analytics/analytics-builder-bundle/getting-started.md [hunk 2]
```diff
@@ -130,19 +130,19 @@ A card for the newly added model is shown on the **Models** tab of the model man
 
 2.  Click the toggle button on the card which currently shows **Inactive**. This changes the state to **Active**.
 
-#### Step 9: Go to the Device Management and view the measurements
+#### Step 9: Go to the Device management and view the measurements
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/getting-started.md:133]:
```suggestion
#### Step 9: Go to the Device management application and view the measurements
```

File: content/users-guide/device-management-bundle/smartrest-templates.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ SmartREST templates are a collection of request and response templates used to c
 
 To add a new template, either import an existing template or create a new one.
 To import a template, click **Import template** at the right of the top menu bar, select a file to be uploaded and provide a name and unique ID.
-To create a new template, click **Create template** at the right of the top menu bar, provide a name and unique ID and add the required messages or responses as described in the *Device Management > SmartREST templates* section in the *User guide*.
+To create a new template, click **Create template** at the right of the top menu bar, provide a name and unique ID and add the required messages or responses as described in the *Device management > SmartREST templates* section in the *User guide*.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/smartrest-templates.md:27]: This should actually not be in italics but be a proper link stead.

**Pull request merged**